### ISSUE TITLE
example doesn't work geneticString.py

### ISSUE THREAD

**Issue by unnir** [mateosss/geneticstring#1]
I git this error message: 

```
  File "//Desktop/gen.py", line 381, in <module>
    rouleteOfGod()
  File "//Desktop/gen.py", line 52, in rouleteOfGod
    for i in range(0,len(population)/2,2):
TypeError: 'float' object cannot be interpreted as an integer
```

it seems you are getting the float for range.

**Comment by mateosss** [comment]
That's pretty weird at first sight, Try using python 3 and python 2.7. I don't remember very well what version I used when I did this. When I have a moment I will check the script to see if I can solve that.

**Comment by unnir** [comment]
I found the issue, you have to use double / for some lines, but to be honest it's weird for me too. 
I'm using Python 3.5.2. 

Anyway, please have a look. Or I can post my code.

**Comment by mateosss** [comment]
The problem was that from python 3, dividing integers produces a float instead of an integer.
So as you said `//` emulates similar behaviour as dividing in python 2.

Also applied autopep8 on the script, it looked very ugly.

Solved in 6058cf9d5675e22ab47bfb81053fe5621e3f54ea (Now works on python2 and python3)
